roles/bootstrap-os/tasks/bootstrap-ubuntu.yml

```diff
@@ -23,7 +23,7 @@
   raw:
     apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y {{ubuntu_packages | join(" ")}}
-  environment: {}
+  environment: "{{ proxy_env }}"
```

Comment:
`environment` attribute is not taken into account by the `raw` module. 
I think `raw` should be replaced here by the `shell` module